### ISSUE TITLE
Remove tedge_apama_plugin

### ISSUE THREAD

**Issue by mjj29** [thin-edge/thin-edge.io#1584]
Hi there, with Apama 10.15 we now ship an Apama plugin built into the Debian packages we ship to support thin-edge.io. For now the package will seamlessly replace the current plugin if it's installed, but you can remove it from your repository at your convenience for any new releases.

**Comment by reubenmiller** [comment]
@mjj29 great, thanks for the information. Though do we still need to support the plugin for installations running older versions, e.g. < 10.15?

**Comment by mjj29** [comment]
Yes you would need to - although prior to 10.15 that was an unsupported as-is offering, with 10.15 it's now officially supported, so I don't think we need to leave it for long - people can be pushed to upgrade to 10.15 if necessary. I'll leave it up to you how/when you remove it
